Source/SessionDelegate.swift

```diff
@@ -83,6 +94,10 @@ extension SessionDelegate: URLSessionTaskDelegate {
         completionHandler(evaluation.disposition, evaluation.credential)
     }
 
+    /// Evaluate the server trust `URLAuthenticationChallenge` received.
```

Comment:
`Evaluate` -> `Evaluates`? Same for other functions as well.

Reply:
Fixed.